uxarray/grid.py

```diff
@@ -528,6 +540,34 @@ def _build_edge_node_connectivity(self):
         setattr(self, "Mesh2_edge_nodes", self.ds['Mesh2_edge_nodes'])
         setattr(self, "nMesh2_edge", edge_nodes_unique.shape[0])
 
+    def _build_face_edges_connectivity(self):
+        """Constructs the UGRID connectivity variable (``Mesh2_face_edges``)
+        and stores it within the internal (``Grid.ds``) and through the
+        attribute (``Grid.Mesh2_face_edges``)."""
+        if "Mesh2_edge_nodes" not in self.ds.keys(
+        ) or (self.ds['Mesh2_edge_nodes'].attrs.get("inverse_indices") is None):
+            self._build_edge_node_connectivity(repopulate=True)
+
+        inverse_indices = self.ds['Mesh2_edge_nodes'].inverse_indices
+        inverse_indices = inverse_indices.reshape(self.nMesh2_face,
+                                                  self.nMaxMesh2_face_nodes)
+        mesh2_face_edges = inverse_indices  # We only need to store the edge index
+
+        self.ds["Mesh2_face_edges"] = xr.DataArray(
+            data=mesh2_face_edges,
+            dims=["nMesh2_face", "nMaxMesh2_face_edges"],
+            attrs={
+                "cf_role":
+                    "face_edges_connectivity",
+                "start_index":
+                    INT_DTYPE(0),
+                "long_name":
+                    "Maps every edge to the two nodes that it connects",
+            })
+
+        # set standardized attributes
+        setattr(self, "nMaxMesh2_face_edges", mesh2_face_edges.shape[1])
+
```

Comment:
We also want to set an attribute for `Mesh2_face_edges`
```suggestion
setattr(self, "Mesh2_face_edges", self.ds["Mesh2_face_edges"])
```